test/RewardsStreamerMP.t.sol

```diff
@@ -1224,21 +1224,24 @@ contract UnstakeTest is StakeTest {
         uint256 secondsLocked = streamer.MIN_LOCKING_PERIOD();
         uint256 reducedStake = 5e18;
         uint256 increasedTime = 365 days;
+        uint256 increasedStake = 2e18;
 
         //initialize memory placehodlders
         uint256[4] memory timestamp;
         uint256[4] memory increasedAccuredMP;
         uint256[4] memory predictedBonusMP;
         uint256[4] memory predictedAccuredMP;
         uint256[4] memory predictedTotalMP;
+        uint256[4] memory predictedTotalMaxMP;
         uint256[4] memory totalStaked;
 
         //stages variables setup
         uint256 stage = 0; // first stage: initialization
         {
             timestamp[stage] = block.timestamp;
             totalStaked[stage] = amountStaked;
-            predictedBonusMP[stage] = _calculateBonusMP(totalStaked[stage], secondsLocked);
+            predictedBonusMP[stage] = totalStaked[stage] + _calculateBonusMP(totalStaked[stage], secondsLocked);
+            predictedTotalMaxMP[stage] = (totalStaked[stage] * streamer.MAX_MULTIPLIER()) + _calculateBonusMP(totalStaked[stage], secondsLocked);
```

Comment:
This is incorrect. `maxMP` should not include the `bonusMP`.
Had a discussion with @gravityblast and @mart1n-xyz about this just a few days ago.

Generally, your `bonusMP` should not affect your `maxMP`. Only your stake amount does.

Reply:
MaxMP is the MAX_MULTIPLIER of _amount deposited. However, when we have bonusMP, the MaxMP would be shifted by the amount of bonus generated. Why this would be incorrect?

Reply:
As mentioned above, I was unsure about it as well and had this checked once and for all (here's the discussion: https://discord.com/channels/864066763682218004/1282974895122419742/1294302914499837963)

Reply:
As discussed offline, there was a misunderstanding (once again) about this.
I need to update this PR accordingly.